app/src/main/kotlin/com/simplemobiletools/keyboard/helpers/Constants.kt

```diff
@@ -45,10 +45,13 @@ const val LANGUAGE_DANISH = 16
 const val LANGUAGE_FRENCH_BEPO = 17
 const val LANGUAGE_VIETNAMESE_TELEX = 18
 
-// keyboard height multiplier options
-const val KEYBOARD_HEIGHT_MULTIPLIER_SMALL = 1
-const val KEYBOARD_HEIGHT_MULTIPLIER_MEDIUM = 2
-const val KEYBOARD_HEIGHT_MULTIPLIER_LARGE = 3
+// keyboard height percentage options
+const val KEYBOARD_HEIGHT_75_PERCENT = 75
+const val KEYBOARD_HEIGHT_90_PERCENT = 90
+const val KEYBOARD_HEIGHT_100_PERCENT = 100
+const val KEYBOARD_HEIGHT_120_PERCENT = 120
+const val KEYBOARD_HEIGHT_140_PERCENT = 140
+const val KEYBOARD_HEIGHT_175_PERCENT = 175
```

Comment:
lets make the steps equal, this looks weird

Reply:
So 100-120-140-160?
I would go for different steps for smaller sizes, since this same step has a bigger effect when using it to reduce size. I would either keep it as is (75-90) or if you prefer same steps I would go for 80-90 and maybe even add 70

Reply:
Let me know if updated values make sense to you.